apps/extension/src/message/popup.ts

```diff
@@ -11,9 +11,10 @@ import { OriginRecord } from '../storage/types';
 export enum PopupType {
   TxApproval = 'TxApproval',
   OriginApproval = 'OriginApproval',
+  Ready = 'PopupReady',
```

Comment:
suggestion: should we rename this to say `Ready = "PopupReadyToAcceptMessages"`?